src/styles/globals.css

```
@@ -175,9 +170,9 @@
   --radius-4xl: calc(var(--radius) + 16px);
 }
 
-@import "./themes/catppuccin.css";
-@import "./themes/t3chat.css";
-@import "./themes/zinc.css";
+@import './themes/catppuccin.css';
+@import './themes/t3chat.css';
+@import './themes/zinc.css';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)